src/test/scala/com/picnicml/doddlemodel/modelselection/CrossValidationTest.scala

```diff
@@ -12,6 +14,7 @@ class CrossValidationTest extends FlatSpec with Matchers {
     (DenseMatrix.zeros[Double](nRows, 1), convert(DenseVector((1 to nRows).toArray), Double))
 
   "3-fold cross validation" should "split 8 examples correctly" in {
+    implicit val rand: Random = new Random(42)
```

Comment:
Ok, I can change the implicit param in splitData to `(implicit rand: Random = new Random())`.

Reply:
No that's fine since it's private. What I meant is that this particular instance of `CrossValidation` doesn't shuffle the rows: https://github.com/novoselrok/doddle-model/blob/deterministic-tests/src/test/scala/com/picnicml/doddlemodel/modelselection/CrossValidationTest.scala#L11, hence the implicit `Random` doesn't have any effect in `CrossValidationTest`.

Reply:
Oh ok, misread your comment. I'll just remove the seed then.